PYXAID2/run_namd.py

```diff
@@ -169,27 +165,22 @@ def compute_L(Phi_basis, Psi_basis, S_adi_ks, S_dia_adi_ks, S_dia_ks):
 def run_namd(params):
 
     rnd = Random()
-
+    T = params["T"]
```

Comment:
please, change the keyword to something more meaningful - did you perhaps mean "Temperature"?